blessedjasonmwanza/hello-rails
# Test/controllers specs

## Test/controllers specs
In this TDD update;
- Install and configure ```Rspec``` and ```rails-controller-testing``` (required for template rendering testing)
- Create test cases that check to ensure ```pages```, ```users ```, ```posts``` controllers meets the following test cases;
  - If the response status was correct.
  - If a correct template was rendered.
  - If the response body includes correct placeholder text.
